assets/css/_sass/website/_layout.scss

```diff
@@ -339,3 +339,23 @@ pre {
     // background-color: $grey-color-dark;
   }
 }
+
+
+/**
+ * Add additional space between hxRow divs for design-system site only
+ */
+
+.hxRow {
```

Comment:
check helix spacing

Reply:
Reverting this change as it will be fixed with https://github.com/rackerlabs/design-system/issues/334